docker/images/test-arm/Dockerfile

```diff
@@ -9,6 +9,7 @@ ADD . $GOPATH/src/github.com/skycoin/libskycoin/
 
 RUN [ "cross-build-start" ]
 
+RUN uname -m
```

Comment:
What for ?

Reply:
it was to know the architecture that was working